### PR TITLE
Add SECURITY.md to the repo

### PR DESCRIPTION
**Pull request by Kowlin**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+vulnerability.
+
+### Opening a Vulnerability Report
+
+To open a vulnerability report please fill out [this form](https://github.com/Cog-Creators/Red-DiscordBot/security/advisories/new)
+
+You will be asked to provide a summary, details and proof of concept for your vulnerability report.
+We ask that you fill out these details to the best of your ability, with as many details as possible.
+Furthermore you'll be asked to provide affected products and severity. These are optional to fill out.
+And will be handled by the maintainers.
```
Comment (Jackenmen) [inline SECURITY.md]: The sentence split on an 'and' feels weird, maybe this is better?
```suggestion
Furthermore, you'll be asked to provide affected products and severity.
These fields are optional and will be filled appropriately by the maintainers if not provided.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+### Timeline
+
+We will try to answer your report within 3 days. If you haven't received a answer we suggest you reach
```
Comment (Jackenmen) [inline SECURITY.md]:
```suggestion
We will try to answer your report within 3 days. If you haven't received an answer by then, we suggest you reach
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+out to us privately. This can best be done via our [Discord server](https://discord.gg/red).
```
Comment (Jackenmen) [inline SECURITY.md]: I'm not sure if this is specific enough, it's a big server and bad actors can be a risk if say, the reporter asks in a public channel who they can contact about a vulnerability.

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Security Policy
+
+## Supported Versions
+
+The table below explains the current state of our versions. Currently only version
```
Comment (Jackenmen) [inline SECURITY.md]:
```suggestion
The table below explains the current state of our versions. Currently, only version
```

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+[here](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)).
+This ensures that all maintainers and key members have access to the reported
+vulnerability.
+
+### Opening a Vulnerability Report
+
+To open a vulnerability report please fill out [this form](https://github.com/Cog-Creators/Red-DiscordBot/security/advisories/new)
+
+You will be asked to provide a summary, details and proof of concept for your vulnerability report.
+We ask that you fill out these details to the best of your ability, with as many details as possible.
```
Comment (Jackenmen) [inline SECURITY.md]:
```suggestion
We ask that you fill out this form to the best of your ability, with as many details as possible.
```

File: SECURITY.md (hunk 3 above)
Comment (Kowlin) [reply SECURITY.md]: Added some more specific text there. Key reason for staff marked is that they'll forward it to the right people.

**Review by Jackenmen — APPROVED** [review]
Let's do this!

**Pull request merged**